output/schema.json

```diff
@@ -6,75 +6,56 @@
   "anyOf": [
     { "$ref": "#/$defs/flag" },
     { "$ref": "#/$defs/basic" },
-    { "$ref": "#/$defs/detailed" },
-    { "$ref": "#/$defs/verbose" }
+    { "$ref": "#/$defs/hierarchical" }
   ],
   "$defs": {
     "outputUnit":{
       "properties": {
         "valid": { "type": "boolean" },
-        "keywordLocation": {
+        "evaluationPath": {
           "type": "string",
           "format": "json-pointer"
         },
-        "absoluteKeywordLocation": {
+        "schemaLocation": {
           "type": "string",
           "format": "uri"
```

Comment:
Shouldn't this be iri now?

Reply:
ah I see you're already on it, re the other PR about uri->iri changes.

Reply:
I'm going to push this forward as I can't wait on your PR any longer.  It needs to be rebased anyway.